README.md

```diff
@@ -63,10 +61,28 @@ When running the scanner with this image, this `.sonar` directory is created in
 
 Caching is actually shared between projects when running the scanner natively as the `.sonar` is created in the home directory of the current user (eg. `/home/my_user/.sonar`).
 
-Here is how you can reproduce this behavior.
+You can reproduce this behavior, by adding the following to the `docker run` command:
+
+```
+-e SONAR_USER_HOME=/usr/.sonar -v "/home/my_user/.sonar:/usr/.sonar"
+```
+
+## Other supported environment variables
+
+### SonarQube authentication
 
-1. specify the new location of the directory in the project's `sonar-project.properties` with `sonar.userHome=/usr/.sonar`
-2. add the following option the `docker run` command: `-v "/home/my_user/.sonar:/usr/.sonar"`
+* `SONAR_TOKEN`: the prefered way to authentication to SonarQube is to use a token. Use this environment variable to pass it to the scanner
+* `SONAR_LOGIN` and `SONAR_PASSWORD`: alternatively, you can provide the SonarQube username and password for the scanner to use using these two variables
+
+### Project mounting point
+
+By default, the scanner analyses the project in directory `/usr/src`.
+
+If you can't mount the project into this directory (eg. in Gitlab CI), use `SONAR_PROJECT_BASE_DIR` to specify another location in the container.
```

Comment:
Should we note that by default if `SONAR_USER_HOME` is not set it will be pointing to the same directory as `SONAR_PROJECT_BASE_DIR`?

Reply:
good point.

I've revamped the doc a bit, grouped all env var doc into the same paragraph and changed order a bit. This allowed me to put a reference to `SONAR_PROJECT_BASE_DIR` in the `SONAR_HOME` part.

Hopefully, this also makes the whole doc clearer

Reply:
Looks fine to me